assets/quick-order-list.css

```diff
@@ -208,12 +188,6 @@ quick-order-list .quantity__button {
   font-weight: 400;
 }
 
-.product-option {
```

Comment:
The errors are not showing up on mobile and there is something odd happening at the top of the PR

This PR: https://screenshot.click/29-09-huql2-rvvvu.mp4

Main: https://screenshot.click/29-11-ciq21-i0vj4.png

Reply:
Works as expected thank you 🎉